File: 코딩 기초 트레이닝/이차원 배열 대각선 순회하기-p181829/이차원 배열 대각선 순회하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181829

def solution(board, k):
    answer = 0

    for i, b in enumerate(board):
        for j, v in enumerate(b):
            if i+j <= k:
                answer += board[i][j]

    return answer

File: 코딩 기초 트레이닝/이차원 배열 대각선 순회하기-p181829/이차원 배열 대각선 순회하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181829

function solution(board, k) {
    var answer = 0;

    board.forEach((b, i) => {
        b.forEach((v, j) => {
            if(i+j <= k) answer += board[i][j];
        });
    });

    return answer;
}
